requirements/run

```diff
@@ -8,7 +8,6 @@ doctr
 rever
 xonsh
 conda-smithy
-requests
```

Comment:
I only see one listing of requests here. Why are you removing this?

Reply:
it's on line 6

Reply:
Omg I am so sorry! Thank you and my apologies!